WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1998,6 +2027,9 @@ private extension EditableOrderViewModel {
         static let newTaxRateSetSuccessMessage = NSLocalizedString("🎉 New tax rate set", comment: "Message when a tax rate is set")
         static let stopAddingTaxRateAutomaticallySuccessMessage = NSLocalizedString("Stopped automatically adding tax rate",
                                                                                     comment: "Message when the user disables adding tax rates automatically")
+        static let customAmountDefaultName = NSLocalizedString("EditableOrderViewModel.customAmountDefaultName",
```

Comment:
Should the initial character not be capitalized despite being a class name? From the examples on p91TBi-aJl-p2

```suggestion
        static let customAmountDefaultName = NSLocalizedString("editableOrderViewModel.customAmountDefaultName",
```

Reply:
Yeah, done in f3a0cd3f533f09a61813bef1282244166b23be7a